src/openrct2/peep/Guest.cpp

```diff
@@ -382,7 +382,7 @@ void Guest::Tick128UpdateGuest(int32_t index)
             PeepThoughtType thought_type = crowded_thoughts[scenario_rand() & 0xF];
             if (thought_type != PEEP_THOUGHT_TYPE_NONE)
             {
-                peep_insert_new_thought(this, thought_type, PEEP_THOUGHT_ITEM_NONE);
+                this->InsertNewThought(thought_type, PEEP_THOUGHT_ITEM_NONE);
```

Comment:
You don't need to do `this->` on a member function when you are already calling it from within a member function.